pylint/extensions/comment_ending.py

```diff
@@ -0,0 +1,56 @@
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IRawChecker
+
+
+def is_line_commented(line):
+    """ Checks if a `# symbol that is not part of a string was found in line"""
+
+    comment_idx = line.find(b"#")
+    if comment_idx == -1:
+        return False
+    if comment_part_of_string(line, comment_idx):
+        return is_line_commented(line[:comment_idx] + line[comment_idx + 1 :])
+    return True
+
+
+def comment_part_of_string(line, comment_idx):
+    """ checks if the symbol at comment_idx is part of a string """
+
+    if (
+        line[:comment_idx].count(b"'") % 2 == 1
+        and line[comment_idx:].count(b"'") % 2 == 1
+    ) or (
+        line[:comment_idx].count(b'"') % 2 == 1
+        and line[comment_idx:].count(b'"') % 2 == 1
+    ):
+        return True
+    return False
+
+
+class CommentChecker(BaseChecker):
+    __implements__ = IRawChecker
+
+    name = "refactoring"
+    msgs = {
+        "R2044": (
+            "Unnecessary comment at end of line",
+            "empty-comment-ending",
```

Comment:
```suggestion
            "empty-comment",
```

Reply:
fixed in` PR review` commit

Reply:
There is still some "ending" in changelog, description and code. I did not comment on all of them but a search for ending should bring all of them :)

Reply:
You are right. Just uploaded a new commit with the fix :D